documentation/docs/index.md

```diff
@@ -2,24 +2,24 @@
 Welcome to the official documentation site for Hummingbot!
 
 !!! warning "Warning: Beta Software"
-    Hummingbot is experimental, open source software that is still in beta, and you use it at your own risk.  Please review the [Apache 2.0 license](https://github.com/CoinAlpha/hummingbot/blob/master/LICENSE).
+    Hummingbot is experimental, open source software that is still in beta, and you use it at your own risk. Please review the [Apache 2.0 license](https://github.com/CoinAlpha/hummingbot/blob/master/LICENSE).
 
```

Comment:
Hummingbot is experimental, open source software that is still in beta; use it at your own risk.